app/src/main/cpp/BrowserWorld.cpp

```diff
@@ -1696,6 +1717,12 @@ BrowserWorld::DrawWorld(device::Eye aEye) {
     m.drawList->Draw(*camera);
   }
 
+  // Draw hand models
+  for (Controller& controller: m.controllers->GetControllers()) {
+    if (controller.enabled && controller.mode == ControllerMode::Hand)
+      m.handModelsRenderer->Draw(*camera, controller);
+  }
```

Comment:
Hmm shouldn't we check that we have the mesh available? I wonder what happens in the case of Pico for example

Reply:
That check is done in `Draw()` method: we bail if `controller.hasHandMesh` is false.